superset/views/base.py

```diff
@@ -66,6 +66,7 @@
     )
 
 FRONTEND_CONF_KEYS = (
+    "ALLOW_DASHBOARD_DOMAIN_SHARDING",
```

Comment:
I don't think you need this here. feature flags always get added into the bootstrap data